client/web/src/site-admin/WebhookCreateUpdatePage.tsx

```diff
@@ -213,17 +213,19 @@ export const WebhookCreateUpdatePage: FC<WebhookCreateUpdatePageProps> = ({ exis
                                 <Input
                                     className={classNames(styles.first, 'flex-1 mb-0')}
                                     message={
-                                        webhook.codeHostKind &&
-                                        webhook.codeHostKind === ExternalServiceKind.BITBUCKETCLOUD ? (
-                                            <small>Bitbucket Cloud doesn't support secrets.</small>
+                                        (webhook.codeHostKind &&
+                                            webhook.codeHostKind === ExternalServiceKind.BITBUCKETCLOUD) ||
+                                        webhook.codeHostKind === ExternalServiceKind.AZUREDEVOPS ? (
+                                            <small>Code Host doesn't support secrets.</small>
```

Comment:
```suggestion
                                        (webhook.codeHostKind &&
                                            (webhook.codeHostKind === ExternalServiceKind.BITBUCKETCLOUD ||
                                        webhook.codeHostKind === ExternalServiceKind.AZUREDEVOPS)) ? (
                                            <small>Code Host doesn't support secrets.</small>
```

Since `webhook.codeHostKind` is nullable, we need the null check for `codehostKind` to be done before comparing.

Reply:
It might be helpful to move this logic into a helper function btw. It's already getting long and a bit hard to read.